modules/optimizing-storage-azure.adoc

```diff
@@ -0,0 +1,12 @@
+:_content-type: REFERENCE
+
+[id="optimizing-storage-azure_{context}"]
+= Optimizing storage performance for Microsoft Azure
+
+{product-title} and Kubernetes are sensitive to disk performance, and faster storage is recommended, particularly for etcd on the control plane nodes. 
+
+For production Azure clusters and clusters with intensive workloads, the VM OS disk should be able to sustain a tested and recommended minimum throughput of 5000 IOPS / 200MBps for control plane machines. 
+This throughput can be provided by having a minimum of 1 TiB Premium SSD (P30). 
+In Azure and Azure Stack Hub, disk performance is directly dependent on SSD disk sizes, so to achieve the throughput supported by `Standard_D8s_v3`, or other similar machine types available, and the target of 5000 IOPS, at least a P30 disk is required.
+
+Host caching must be set to `ReadOnly` for low read latency and high read IOPS and throughput. The reads performed from the cache, which is present either in the VM memory or in the local SSD disk, are much faster than the reads from the data disk, which is in the blob storage.
```

Comment:
Can you replace "reads" with a different noun? You shouldn't use verbs as nouns (or nouns as verbs).

Reply:
Reworded to use "reading data" instead of "reads".